app/admin/pages.rb

```diff
@@ -261,7 +262,14 @@
               para "Year: #{component&.published_on_year}" if component&.published_on_year.present?
               para "Authors: #{component&.authors}" if component&.authors.present?
             end
-
+            # Event
+            if pc.component_type == 'PageEventComponent'
+              ul do
+                component.class::FORM_FIELDS.each do | key, value|
+                  li "#{value}: #{component.send(key)}" if component.send(key).present?
+                end
```

Comment:
Threw some bullet points on here for readability
![Screenshot 2023-07-31 at 2 27 22 PM](https://github.com/department-of-veterans-affairs/diffusion-marketplace/assets/5402927/fedaadae-4ca1-4530-ac7e-3e7b49e33652)